migrations/2021-04-16-130927_language-tags/up.sql

```diff
@@ -2,5 +2,5 @@ ALTER TABLE comment ADD COLUMN language TEXT NOT NULL;
 ALTER TABLE post ADD COLUMN language TEXT NOT NULL;
 ALTER TABLE private_message ADD COLUMN language TEXT NOT NULL;
 
-ALTER TABLE local_user ADD COLUMN discussion_languages TEXT[] NOT NULL;
+CREATE TABLE discussion_languages(id INTEGER PRIMARY KEY, local_user_id INT NOT NULL, language TEXT NOT NULL);
```

Comment:
This isn't correct, look at the other create tables in this folder. The id column is wrong and its not doing a foreign key reference.

Reply:
I think I have only done this kind of thing in sqlite and maybe mysql, had no idea that postgres handles it that much nicer.